src/ecrc-frontend/src/components/page/success/Success.js

```diff
@@ -187,7 +187,8 @@ export default function Success({
       })
       .then(urlResponse => {
         window.location.href = urlResponse.data.paymentUrl;
-      });
+      })
+      .catch(() => {});
```

Comment:
why not take to error page here as well?

Reply:
I was having some issues with error handling on this page that I was having a tough time figuring out. @adodge-bcg is going to take a look at it later.